content/en/docs/ops/ambient/getting-started/index.md

```diff
@@ -6,5 +6,255 @@ weight: 1
 owner: istio/wg-networking-maintainers
 test: n/a
 ---
+{{ warning }}
+Ambient is in alpha status, use at your own risk.
+{{ /warning }}
 
-This page is under construction.
+This guide lets you quickly evaluate Istio ambient service mesh. These steps require you to have
+a {{< gloss >}}cluster{{< /gloss >}} running a
+[supported version](/docs/releases/supported-releases#support-status-of-istio-releases) of Kubernetes ({{< supported_kubernetes_versions >}}). You can use any supported platform, for
+example [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) or
```

Comment:
Is minikube really good enough? I thought minikube only allows one external lb service. IIUC, waypoint proxies use k8s gateway, each one of which will need an external lb.

Reply:
Yes, because waypoint proxy don't need external LB service.